core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -417,22 +566,34 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
     }
   }
 
-  private class InternalRebalanceListener(connector: ZookeeperConsumerConnector,
-                                          customRebalanceListener: Option[ConsumerRebalanceListener])
-    extends ConsumerRebalanceListener {
+  private class InternalRebalanceListener(mirrorMakerConsumer: MirrorMakerBaseConsumer,
```

Comment:
Why do we need to merge ```org.apache.kafka.clients.consumer.ConsumerRebalanceListener``` into ```InternalRebalanceListener```? Could just create the old / new consumers AND initialize them with the different types of listeners separately and pass them to extensions of ```MirrorMakerBaseConsumer```?